src/main/java/org/broadinstitute/hellbender/tools/walkers/longreads/QuicklyCountMismatches.java

```diff
@@ -0,0 +1,138 @@
+package org.broadinstitute.hellbender.tools.walkers.longreads;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.ExperimentalFeature;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.utils.pileup.PileupElement;
+import picard.cmdline.programgroups.ReadDataManipulationProgramGroup;
+import org.broadinstitute.hellbender.exceptions.UserException;
+
+import java.io.FileNotFoundException;
+import java.io.PrintStream;
+import java.util.*;
+
+/**
+ * Quickly count errors
+ *
+ * <h3>Input</h3>
+ * <ul>
+ *     <li>A single BAM file</li>
+ * </ul>
+ *
+ * <h3>Outputs</h3>
+ * <ul>
+ *     <li>A collection of BAM files where care has been keep reads from the same ZMW in the same file</li>
+ * </ul>
+ *
+ * <h3>Usage Example</h3>
+ * <h4>Quickly count errors</h4>
+ * <pre>
+ *   gatk QuicklyCountMismatches \
+ *     -I input.bam \
+ *     -O output.txt
```

Comment:
It looks to me that all 3 outputs are required but you haven't included those in the outputs. Furthermore it is confusing to have -O -insHist and -delHist. I would recommend either making an output directory, or chaning -O to -mismatchHist